crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -751,7 +747,7 @@ impl PipelineCache {
                     multiview: None,
                     depth_stencil: descriptor.depth_stencil.clone(),
                     label: descriptor.label.as_deref(),
-                    layout: layout.as_deref(),
+                    layout: layout.as_ref().map(|layout| -> &PipelineLayout { layout }),
```

Comment:
This is a bit icky but it's the only way I could think of to coerce the `Arc<WgpuWrapper<PipelineLayout>>` to `&PipelineLayout`

Reply:
i think `layout.as_ref().map(|v| &***v)` (or `layout.as_deref().map(|v| &**v)`) is marginally clearer.

Reply:
Done

Reply:
Your suggestion doesn't work, since `WgpuWrapper` has a different type signature between wasm32 and everything else, so your specific number of derefs `&**` only works for the non-wasm32-version.

My version works reliably for both, because the compiler will insert the correct number of derefs at compile time.

What do you want me to do?

Reply:
Applied the original fix